converters/fromZigbee.js

```diff
@@ -2488,6 +2488,17 @@ const converters = {
             }
         },
     },
+    easycodetouch_action: {
+        cluster: 'closuresDoorLock',
+        type: 'raw',
+        convert: (model, msg, publish, options, meta) => {
+            const value = constants.easyCodeTouchActions[(msg.data[3] << 8) | msg.data[4]];
+            if (!value) {
+                throw new Error('Unknown lock status with source ' + msg.data[3] + ' and event code ' + msg.data[4]);
```

Comment:
Instead of throwing an error, please do ` debug.log`

Reply:
Now using meta.logger.log